src/list-data/request-action-mapping.ts

```diff
@@ -98,12 +98,22 @@ export const BusinessLookupRequestActions = [
 
 /** Entity types that require business lookup. */
 export const BusinessLookupEntityTypes = [
+  EntityTypes.A,
   EntityTypes.BC,
+  EntityTypes.BEN,
   EntityTypes.CC,
+  EntityTypes.CP,
   EntityTypes.CR,
-  EntityTypes.UL,
+  EntityTypes.GP,
+  EntityTypes.LLC,
+  EntityTypes.LP,
+  EntityTypes.SP,
+  EntityTypes.ULC,
+  EntityTypes.XCP,
   EntityTypes.XCR,
-  EntityTypes.CP,
+  EntityTypes.XL,
+  EntityTypes.XP,
+  EntityTypes.XS,
```

Comment:
I'm wondering if this should be `XSO` ? This is also used in a couple of places in the code that might be wrong. I don't know how to answer this question. Linda might know but I think she's still away. Maybe ask Mihai or someone from the business?

Reply:
I just talked with Mihai. He provided the attached list of name request types, which show XSO but not SO.

Mihai also says we can just add both codes.

Therefore:
1. please add both codes to this list
2. please change the 2 places in code where it uses "XS" to be "XSO" instead

[name request types.txt](https://github.com/bcgov/namerequest/files/12678205/name.request.types.txt)